### PR TITLE
packs/reco: change path of url

### PR DESCRIPTION
**Pull request by GalNakash-RecoLabs**
Change alerts path

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by GalNakash-RecoLabs** [comment]
@sapirshuker let me know if anything is missing

**Comment by sapirshuker** [comment]
Hey, @GalNakash-RecoLabs 
Hi @contributor_user,
Thank you for your contribution. 
I will review your PR soon. 
In the meantime, you can try to solve the errors in the [build](https://app.circleci.com/jobs/github/demisto/content/724355).
I'm available for any of your questions here or on slack.
Thanks again.

File: Packs/Reco/ReleaseNotes/1_1_7.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Reco
+
+- Change alerts URL to point to the new alerts endpoint
```
Comment (sapirshuker) [inline Packs/Reco/ReleaseNotes/1_1_7.md:6]:
```suggestion
- Updated the URL endpoint to the new alerts endpoint in commands:
    - ***reco-change-alert-status***
    - ***fetch_incidents***
```
Verify that these are all the commands that have been changed.

File: Packs/Reco/Integrations/Reco/Reco.py [hunk 2]
```diff
@@ -191,7 +191,7 @@ def get_alerts(
         try:
             response = self._http_request(
                 method="PUT",
-                url_suffix="/alert-inbox/table",
+                url_suffix="/policy-subsystem/alert-inbox/table",
```
Comment (sapirshuker) [inline Packs/Reco/Integrations/Reco/Reco.py:194]: What is the effect of this change on backward compatibility?

**Review by sapirshuker — CHANGES_REQUESTED** [review]
Hey, thank you for your contribution. 
The code looks good!
Please solve the errors in the [build](https://app.circleci.com/jobs/github/demisto/content/724355)

File: Packs/Reco/Integrations/Reco/Reco.py (hunk 2 above)
Comment (GalNakash-RecoLabs) [reply Packs/Reco/Integrations/Reco/Reco.py:194]: No backward compatibility - we changed it

**Comment by sapirshuker** [comment]
Hey, @GalNakash-RecoLabs 
I would be happy to know why you did this change?
Does the new endpoint match the old endpoint?
Do the results from the endpoint align with the results from the old endpoint? (the results is the same for both)
Thanks

**Comment by sapirshuker** [comment]
Hey, @GalNakash-RecoLabs, the code looks good!

We're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR.

**Comment by GalNakash-RecoLabs** [comment]
I'm in miluim.
I will record a demo over the weekend and share it with you.
Best,
Gal

Sent via Superhuman iOS ( ***@***.*** )

**Comment by sapirshuker** [comment]
Hey, @GalNakash-RecoLabs 
Thank you for the demo.
Please merge from master in order to continue the process.

**Review by sapirshuker — APPROVED** [review]
Great work!

**Pull request merged**